### DISCUSSION TITLE
What does Objeck's web framework look like?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#105] [General]
As you said on https://github.com/objeck/objeck-lang/discussions/85 it's based on XML. I can't find any XML on https://github.com/objeck/objeck-lang/tree/master/programs/web Could you tell me how it look like? It's most similar to which web framework on Java world? Thank you.

**Comment by objeck** [comment]
I am working on a new [web frame](https://github.com/objeck/objeck-lang/blob/master/core/compiler/lib_src/web_server.obs). Starting with [IIS](https://github.com/objeck/objeck-lang/tree/master/core/lib/web/iis) for Windows, followed by [Nginx](https://github.com/objeck/objeck-lang/tree/master/core/lib/web/nginx) for Linux, last Apache.

**Reply by ghost** [reply]
Is it still XML based?

**Comment by objeck** [comment]
The previous web framework had a markup language based on XHTML and inline Objeck code. I will probably reuse and extend that solution.

**Comment by objeck** [comment]
The near-term focus is to move away from FastCGI, which is unmaintained, and build native webserver modules for Nginx, IIS, and Apache. In addition, there will be a standard abstraction layer so code can be run on different web servers without being rewritten.
